### PR TITLE
Created norwegian translation (no.yml)

### PR DESCRIPTION
**Pull request by Fraithor**

### CODE REVIEW

File: src/main/resources/no.yml [hunk 1]
```diff
@@ -0,0 +1,62 @@
+# Hjelp med å oversette denne filen til flere språk!
+Messages:
+  Prefix: '&7 [&6BetterRTP &7]'
+  Success: ## Plassholdere! %x% %y%og %z% er koordinatene x, y og z som spilleren blir teleportert til! #
+    Paid: '&aDu har blitt teleportert til &7 x=%x%, y=%y%, z=%z%, for &c $%price%&7 i &f%forsøk%&7 forsøk!'
+    Bypass: '&aYou have been teleportert to &7 x=%x% y=%y% z=%z% etter &f%forsøk%&7 forsøk'
```
Comment (Janbchr) [inline src/main/resources/no.yml:6]: Not translated yet

File: src/main/resources/no.yml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  Edit: 
+    Error: '&cError! &7Ugyldig verdi er gitt! '
+    Set: '&bSuksess! &7%type% satt til %verdi%'
+    Remove: '&cFjernet! &7Du fjernet egendefinert verden %world%'
+
+Help:
+  Prefix: '&e &m ----- &6 &l BetterRTP &8 | Hjelp &e &m ----- '
+  Main: '&7- &e /%command%&7- Teleporterer deg tilfeldig!'
+  Biome: '&7- &e /%command% biome <biome1, biome2 ...> &7- Tilfeldig teleportering med disse biomene'
+  Edit: '&7- &e /%command% edit <default / world> [args ...] &7- Edit some plugin settings'
```
Comment (Janbchr) [inline src/main/resources/no.yml:40]: Not translated yet

File: src/main/resources/no.yml [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+  Help: '&7- &e /%command% help &7- viser hjelpeliste'
+  Info: '&7- &e /%command% info [verden / partikler / former / potion_effects] &7- Vis spesifikk informasjon om plugin-parametere'
+  Player: '&7- &e /%command% player <spiller> [verden] [biome1, biome2 ...] &7- Tilfeldig teleportere en annen spiller'
+  Reload: '&7- &e /%command% reload &7- Last inn plugin'
+  Settings: '&7- &e /%command% settings &7- Trekk opp en gui og rediger noen innstillinger'
+  Test: '&7- &e /%command% test &7- Test ut plugin-effekter etter en teleport uten å bevege seg'
+  Version: '&7- &e /%command% version &7- Vis for øyeblikket kjørende versjon'
+  World: '&7- &e /%command% world <world> [biome1, biome2 ...] &7- Tilfeldig teleportering i en annen verden'
+  Location: '&7- &e /%command% location <location_name> &7- Rtp using a specific location'
```
Comment (Janbchr) [inline src/main/resources/no.yml:49]: Not translated yet

File: src/main/resources/no.yml [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+Usage:
+  Player: '&cUsage &7: /%command% player <player> [world] [biome1, biome2]'
```
Comment (Janbchr) [inline src/main/resources/no.yml:52]: In line 52 - 62 the word "Usage" is not translated

Comment (Dragohuntrrr) [reply src/main/resources/no.yml:52]: But, I thought that the path should be in english, while the value is in the translated language

File: src/main/resources/no.yml (hunk 1 above)
Comment (Janbchr) [reply src/main/resources/no.yml:6]: Bypass: '&a**You have been** teleportert **to** &7 x=%x% y=%y% z=%z% etter &f%forsøk%&7 forsøk'

File: src/main/resources/no.yml (hunk 2 above)
Comment (Janbchr) [reply src/main/resources/no.yml:40]: Edit: '&7- &e /%command% edit <default / world> [args ...] &7- **Edit some plugin settings**'

File: src/main/resources/no.yml (hunk 3 above)
Comment (Janbchr) [reply src/main/resources/no.yml:49]: Location: '&7- &e /%command% location <location_name> &7- **Rtp using a specific location**'

File: src/main/resources/no.yml (hunk 4 above)
Comment (Janbchr) [reply src/main/resources/no.yml:52]: E.g. Player: '&c**Usage** &7: /%command% player <player> [world] [biome1, biome2]'

It is simply just the word "Usage" in each line that needs translation.

**Comment by Dragohuntrrr** [comment]
Thank you, uploaded new no.yml

**Comment by SuperRonanCraft** [comment]
thank you all for the new language file :)

**Pull request merged**